### ISSUE TITLE
Desktop client

### ISSUE THREAD

**Issue by trymeouteh** [dreautall/waterfly-iii#46]
Please compile and release this app for desktop.

**Comment by much-doge** [comment]
i agree, in desktop the web interface is better and also more powerful as it is the main frontend of firefly

also, these app for every service is just feels wrong in desktop/pc when there is  a functional web interface
